buildlib/tests.yml

```diff
@@ -40,4 +40,8 @@ jobs:
           JENKINS_TEST_PERF: ${{ parameters.test_perf }}
           JENKINS_NO_VALGRIND: ${{ parameters.valgrind_disable }}
           RUNNING_IN_AZURE: yes
-
+  - template: jucx/jucx-test.yml
+    parameters:
+      name: ${{ parameters.name }}
+      demands: ${{ parameters.demands }}
+      worker_id: $(worker_id)
```

Comment:
Here worker_id is not define in this section

if you want use several launch you should use `strategy` like in `- job: tests_${{ parameters.name }}`

```
  - template: jucx/jucx-test.yml

    strategy:
      matrix:
        ${{ each wid in parameters.worker_ids }}:
          ${{ wid }}:
            worker_id: ${{ wid }}
    parameters:
      name: ${{ parameters.name }}
      demands: ${{ parameters.demands }}
      worker_id: $(worker_id)
```
like this